packages/url_launcher/lib/url_launcher.dart

```diff
@@ -20,9 +20,10 @@ const MethodChannel _channel = MethodChannel('plugins.flutter.io/url_launcher');
 /// [forceSafariVC] is only used in iOS. If unset, the launcher opens web URLs
```

Comment:
To make it a tiny bit more obvious, maybe: `By default (when unset), the launcher opens HTTP/HTTPS URLs`...